packages/venia-ui/lib/components/Header/storeSwitcher.js

```diff
@@ -7,7 +7,7 @@ import { useStoreSwitcher } from '@magento/peregrine/lib/talons/Header/useStoreS
 import { mergeClasses } from '../../classify';
 import defaultClasses from './storeSwitcher.css';
 import SwitcherItem from './switcherItem';
-import storeSwitcherOperations from './storeSwitcher.gql';
+import storeSwitcherOperations from '@magento/peregrine/lib/talons/Header/storeSwitcher.gql';
```

Comment:
You no longer need to pass these operations to the talon since the talon imports and defaults to them when nothing is passed. You can remove this import and the passed arg on line 14-16.